tensorflow/python/keras/layers/simplernn_test.py

```diff
@@ -43,6 +43,8 @@ def test_return_sequences_SimpleRNN(self):
         input_shape=(num_samples, timesteps, embedding_dim))
 
   def test_float64_SimpleRNN(self):
+    if test.is_built_with_rocm:
```

Comment:
Shouldn't this be `is_built_with_dml` and shouldn't this be a function call?

Reply:
Nothing to see here folks. Move along.